docs/docs_build.py

```diff
@@ -86,6 +86,7 @@ def create_version(self, version: Version) -> None:
         os.chdir("..")  # TODO: none of this messing with current directory stuff
         prepend_version_info_to_name_for_snippet_by_name_references(version)
         os.chdir("docusaurus")
+        self._write_release_version(old_version_file)
```

Comment:
Stray fix - sorry for the scope creep. See above for where we store the old version, then temporarily overwrite it.